setup.py

```diff
@@ -92,6 +92,7 @@ def run(self):
     include_package_data=True,
     use_scm_version=True,
     distclass=Distribution,
+    python_requires='>3.7.0',
```

Comment:
Do you need '>=3.7.0' instead of '>3.7.0' here?

Reply:
yes! thank you!